workshops/1.md

```diff
@@ -57,64 +58,68 @@ We agreed on disabling GitHub issues for internal projects and moving all issue
 
-We agreed that if it can't be reproduced then is not a bug.
+We agreed that if it can't be reproduced, then is not a bug.
 
-We agreed that being aware of when the customer is going to demo the projects is critical for better planning on our side. When it will be made, which devices will be used, etc.
+We agreed that being aware of when the customer is going to demo the projects is critical for better planning on our side. When it will be made, which devices will be used, etc. Also, that developers should be consulted on timeline of deliverables. It's really hard to estimate something, and people always understimate development time. This ends up souring relationships with clients, as deadlines are not met and projects get delayed.
 
-We agreed to only work with our own backends, and if we're forced to use an external backend use a middle layer that eases some of the pain with dealing with external APIs.
+We agreed to only work with our own backends whenever possible. If we're forced to use an external backend, to at least use a middle layer that eases some of the pain from dealing with poorly designed or hard to change external APIs.
 
-Finally we need to *always* unit test the API calls, so far we haven't had a good experience working with APIs so this might change in the future if things change, for as for now, Gotta Test 'Em all.
+Finally we need to *always* unit test the API calls, so far we haven't had a good experience working with APIs, so this might change in the future, but for now, we _Gotta Test 'Em All_.
 
 ### Collaboration with designers
 
-We need to be aware of anti-patterns. Basically UI choices that are too fragile and could easily break with the next iOS update. We work with a lot of startups and companies that will maintain our code, so it's important for us to keep things in the native side as much as possible to avoid issues for them. In other words if it involves a hack, then it's probably a bad idea.
+We need to be aware of anti-patterns, UI choices that are too fragile and could easily break with the next iOS update. Diverging needlessly from the platform language. Adding too much visual fluff that takes a lot of time to develop and has very little pay-off. We should probably limit the amout of wowing fluff to one or two per project, at most. We work with a lot of startups and companies that will maintain our code in the future, so it's important for us to keep things in the native side as much as possible, to avoid future issues for them. In other words, if it involves a hack or going against the platform APIs, then it's probably a bad idea.
 
 A few other common anti-patterns:
-- Form fields without labels
-- Text only buttons
-- Square buttons (rounded corners please)
+- Form fields without labels.
+- Text only or textless buttons (with exceptions).
+- Square buttons (rounded corners please).
+- `X` for closing modals.
```

Comment:
I would say this "with exceptions". For a photo viewer having an "X" I think it's fine, for example.